src/VisualStudio/TestUtilities/Remoting/WorkspaceWrapper.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.CodeAnalysis.Editor.Options;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.VisualStudio.LanguageServices;
+
+namespace Roslyn.VisualStudio.Test.Utilities.Remoting
+{
+    internal class WorkspaceWrapper : MarshalByRefObject
+    {
+        private readonly VisualStudioWorkspace _workspace;
+
+        public static WorkspaceWrapper Create()
+        {
+            var visualStudioWorkspace = RemotingHelper.VisualStudioWorkspace;
+            return new WorkspaceWrapper(visualStudioWorkspace);
+        }
+
+        private WorkspaceWrapper(VisualStudioWorkspace workspace)
+        {
+            _workspace = workspace;
+        }
+
+        public bool UseSuggestionMode
```

Comment:
Because it is easier to expose it directly through the WorkspaceWrapper than to create an additional OptionSetWrapper or OptionServiceWrapper that exposes the property itself